### PR TITLE
fix: fix UI issues from sprint review [AR-993][AR-995]

### PR DESCRIPTION
**Pull request by gongracr**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-993" title="AR-993" target="_blank">SubtaskAR-993</a>  Investigate UI scaling issues
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues
Some UI elements on the Home Screen were not correct, hence incorrect, namely padding values, fonts, buttons and images sizes. Adapted them to follow the UI specs.

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/ui/common/Logo.kt [hunk 1]
```diff
@@ -19,8 +20,8 @@ fun Logo() {
         contentDescription = stringResource(id = R.string.app_logo_description),
         contentScale = ContentScale.Fit,
         modifier = Modifier
-            .width(80.dp)
-            .height(30.dp)
-            .padding(8.dp)
+            .padding(paddingValues = PaddingValues(horizontal = 8.dp, vertical = 32.dp))
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/ui/common/Logo.kt:23]: is there a specific reason why you use PaddingValues ? you can just do it like this : padding(horizontal = 8.dp, vertical = 32.dp), also those .dp to dimensions file

File: app/src/main/kotlin/com/wire/android/ui/common/Logo.kt [hunk 2]
```diff
@@ -19,8 +20,8 @@ fun Logo() {
...
+            .width(72.dp)
+            .height(23.dp)
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/ui/common/Logo.kt:25]: move to dimensions, also 23.dp is afunny size :P maybe 24.dp would make more sense as it is multiplier of 8 :P

File: app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt [hunk 3]
```diff
@@ -65,7 +68,7 @@ fun SearchBarUI(placeholderText: String, modifier: Modifier = Modifier, onTextTy
                     }) {
                         Icon(
                             modifier = Modifier.height(35.dp),
-                            imageVector = Icons.Filled.Close,
+                            painter = painterResource(id = R.drawable.ic_clear_search),
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt:71]: is this icon going to be specific for clearing the search only  ? or could it be reused ? if yes, maybe naming it to ic_clear would be better

File: app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt [hunk 4]
```diff
@@ -49,17 +49,11 @@ fun HomeDrawer(
 
     Column(
         modifier = Modifier
-            .padding(top = 40.dp, start = 8.dp, end = 8.dp, bottom = 16.dp)
+            .padding(top = 0.dp, start = 8.dp, end = 8.dp, bottom = 16.dp)
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt:52]: no need to give it 0.dp it is done by default

File: app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt [hunk 5]
```diff
@@ -65,7 +68,7 @@ fun SearchBarUI(placeholderText: String, modifier: Modifier = Modifier, onTextTy
                     }) {
                         Icon(
                             modifier = Modifier.height(35.dp),
-                            imageVector = Icons.Filled.Close,
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt:68]: move to dimens

File: app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt [hunk 6]
```diff
@@ -47,8 +49,9 @@ fun SearchBarUI(placeholderText: String, modifier: Modifier = Modifier, onTextTy
         leadingIcon = {
             IconButton(modifier = Modifier.height(40.dp), onClick = {}) {
                 Icon(
-                    Icons.Filled.Search,
-                    stringResource(R.string.content_description_clear_content)
+                    painter = painterResource(id = R.drawable.ic_search_icon),
+                    contentDescription = stringResource(R.string.content_description_conversation_search_icon),
+                    tint = MaterialTheme.colorScheme.onBackground
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt:54]: no need to do that if we follow the default design values for the IconButton, under the hood IconButton provides the correct color, if it does not suit your needs than   tint = MaterialTheme.colorScheme.onBackground would make sense, do not know your use case

File: app/src/main/kotlin/com/wire/android/ui/common/Logo.kt (hunk 2 above)
Comment (gongracr) [reply app/src/main/kotlin/com/wire/android/ui/common/Logo.kt:25]: Those are the dimensions provided by the design specs 🙈 but will bump them to prettier numbers 😉

File: app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt (hunk 6 above)
Comment (gongracr) [reply app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt:54]: Thanks, I should have specified `MaterialTheme.wireColorScheme.onBackground`. Good catch!

**Review by trOnk12 — APPROVED** [review]
lgtm <3

File: app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt [hunk 7]
```diff
@@ -99,11 +98,12 @@ fun DrawerItem(data: DrawerItemData, selected: Boolean, onItemClick: () -> Unit)
     Row(
         verticalAlignment = Alignment.CenterVertically,
         modifier = Modifier
+            .padding(bottom = 8.dp)
             .clip(RoundedCornerShape(12.dp))
             .fillMaxWidth()
             .height(40.dp)
             .background(backgroundColor)
-            .selectableBackground(selected) { onItemClick() }
+            .selectableBackground(selected) { onItemClick() },
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt:106]: Nothing blocker, but I think it's not necessary that comma at the end

**Pull request merged**